CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
OK, where is your Pyhton.h? Maybe we don't set the correct include directory. Can you check in CMakeCache.txt the value of PYTHON_INCLUDE_DIR? For debugging you can also change into "build/src/system/libs/swig" and run "VERBOSE=1 make" to see the compiler call. It should include the "-I" for the Python include directory.